src/routes/tools/local-climate-change-snapshot/explore.svelte

```diff
@@ -88,6 +92,16 @@
   $: console.log("chart data", $chartDataStore);
   $: console.log("snapshot data", $snapshotDataStore);
 
+  function handleHeaderBtnClick(event) {
+    if (event.target.dataset.action === "change-location") {
+      // Note: using this method of page navigation as there appears to be a bug
```

Comment:
change to `// Note we're using...` and add `this may have to do with explore.svelte fetching index.json.js`